articles/realtime.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: default
+title: ROS 2.0 in Realtime
+permalink: articles/realtime.html
+abstract: This article describes real-time computing requirements and how ROS 2 will fulfill them.
+published: false
+author: Jackie Kay
+---
+
+* This will become a table of contents (this text will be scraped).
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+Robotic systems need to be responsive.
+In mission critical applications, a delay of less than a millisecond in the system can cause a catastrophic failure.
+For ROS 2.0 to capture the needs of the robotics community, the core software components must not interfere with the requirements of real-time computing.
+
+# Definition of Real-time Computing
+
+Real-time software guarantees correct computation at the correct time.
```

Comment:
Maybe introduce key terms in the definition, like *determinism* and *deadline*. Something like:

real-time software meets computational *deadlines* ina *deterministic* way. The correctness depends not only on the result of a computation, but also on the time it was delivered. Thus, failure to respond is as bad as the wrong response.

Paraphrased from http://www.cse.unsw.edu.au/~cs9242/08/lectures/09-realtimex2.pdf